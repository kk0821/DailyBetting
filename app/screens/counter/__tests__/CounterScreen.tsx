import * as React from 'react';
import { shallow } from 'enzyme';
import { CounterScreen } from '../';

let navigation: any;

beforeEach(() => {
  navigation = { navigate: jest.fn() };
});

it('renders correctly', () => {
  const screen = shallow(<CounterScreen count={1} navigation={navigation} />);
  expect(screen).toMatchSnapshot();
});
