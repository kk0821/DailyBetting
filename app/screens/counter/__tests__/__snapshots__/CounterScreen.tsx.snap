// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component
  count={1}
  onDecrement={[Function]}
  onIncrement={[Function]}
/>
`;
